dts/bindings/pinctrl/nordic,nrf-pinctrl.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    nRF pin controller
+
+compatible: "nordic,nrf-pinctrl"
+
+include:
+    - name: base.yaml
+    - name: pincfg-node-group.yaml
+      child-binding:
+        child-binding:
+          property-allowlist:
+            - bias-disable
+            - bias-pull-down
+            - bias-pull-up
+            - low-power-enable
+
+child-binding:
+  description: |
+    nRF pin controller pin configuration nodes. Each node is composed by one or
+    more groups, each defining the configuration for a set of pins.
+
+  child-binding:
+    description: |
+      nRF pin controller pin configuration group. Each group contains a list of
```

Comment:
I think this may be clearer if it appears in the parent description (the one for the child binding, not the grandchild binding).

Reply:
moved all details to the parent description